src/node/BrowserRunner.ts

```diff
@@ -36,6 +36,7 @@ Please check your open processes and ensure that the browser processes that Pupp
 If you think this is a bug, please report it on the Puppeteer issue tracker.`;
 
 export class BrowserRunner {
+  private _product: string;
```

Comment:
We already have a type for this in common/Product.js. Can you use that one instead of string?

Reply:
Oh, I totally overlooked that `Launcher.product` already returns that type. Will absolute fix that.